### ISSUE TITLE
Rachni Soldiers don't come out of vents in Mira Core

### ISSUE THREAD

**Issue by henbagle** [henbagle/LE1CommunityPatch#58]
**Describe the bug**
In certain scenarios, the rachni soldiers do not come out of the vents in the Mira Core. After coming back from the landlines and from the power plant, a single rachni soldier is supposed to come out of a vent each time. In certain situations, the vent will open but the rachni does not appear. 

**To Reproduce**
This bug doesn't seem to always happen. The easiest way I can reproduce it is by loading a save in the Mira Core after coming back from the landlines. Playing the section of game through without loading a save does not trigger the bug, but loading a save in the mira core before triggering the vent will typically cause the rachni to not appear.

**Screenshots & Saves**
[Alden00_156.zip](https://github.com/henbagle/LE1CommunityPatch/files/7157744/Alden00_156.zip)

**Game file**
BIOA_ICE50_06_DSG.pcc. Either something in sequence export 7710, or something to do with the BioPawns, exports 1705 and 1707.

I've looked into this issue a bit and added some logging. At first I thought the Rachni's "Active" property was being set to false, but reactivating the Rachni did not solve the issue. Based on the issue coming from a savegame, my assumption is that some state is being set that causes the issue on a load, but not on a fresh playthrough of the level.

**Comment by henbagle** [comment]
Bug may also be an issue in the attack after taking the trams

**Comment by henbagle** [comment]
Huge thanks to Kinkojiro for helping with this one! The bug seems to be caused because the BIOA_ICE50_06_DSG file does not get unloaded on the roof. Due to this, the sequencing to initiate the Rachni in the vent does not get re-run when returning from the landlines, and so the Rachni remains disabled from the player's initial visit to the Mira Core. 

To fix this, I have altered the streaming in BIOA_ICE00.pcc so that the BIOA_ICE50_06 files are unloaded when the player is on the roof. Specifically, I have removed the LoadedChunkNames property from the BioStreamingState in export 548 in BIOA_ICE00. This causes the ICE50_06 file to unload when the player leaves the hallway and enters the roof area. This doesn't seem to cause any other issues, but more testing is needed.
![image](https://user-images.githubusercontent.com/8151477/142735628-46370ef9-7672-4736-a71c-5fb7f7573110.png)

Any saves that already exist and trigger this bug could still have the issue.
